src/components/pages/GlobalSettings/General/ConnectCloud.tsx

```diff
@@ -1,49 +1,48 @@
-import {Button, Space} from 'antd';
+import {Button} from 'antd';
 
 import {CloudUploadOutlined} from '@ant-design/icons';
 
 import {ExternalLink} from '@atoms';
 
-import {Text} from '@custom-antd';
+import {FullWidthSpace, Text} from '@custom-antd';
 
 import {ConfigurationCard} from '@molecules';
 
+import {openOSSToCloudMigrateLink} from '@utils/externalLinks';
+
 import Colors from '@styles/Colors';
 
-const ConnectCloud = () => {
+const ConnectCloud: React.FC = () => {
   return (
     <ConfigurationCard
       title="Connect to Testkube Cloud"
       description="Connect your current Testkube instance to Testkube Cloud. You can always safely disconnect."
       footerText={
-        <Text className="regular middle" color={`${Colors.slate400}`}>
+        <Text className="regular middle" color={Colors.slate400}>
           Learn more about{' '}
           <ExternalLink href="https://docs.testkube.io/testkube-cloud/articles/transition-from-oss">
             connecting to Testkube Cloud
           </ExternalLink>
         </Text>
       }
     >
-      <Space size={32} direction="vertical" style={{width: '100%', textAlign: 'center'}}>
-        <Space size={20} direction="vertical">
-          <CloudUploadOutlined style={{fontSize: '50px', color: `${Colors.slate500}`}} />
+      <FullWidthSpace size={32} direction="vertical" style={{textAlign: 'center'}}>
+        <FullWidthSpace size={20} direction="vertical">
+          <CloudUploadOutlined style={{fontSize: '50px', color: Colors.slate500}} />
           <Text className="regular middle">Get more out of Testkube</Text>
-          <Space size={1} direction="vertical">
-            <Text className="regular middle" color={`${Colors.slate400}`}>
+          <FullWidthSpace size={1} direction="vertical">
```

Comment:
Should it be `FullWidthSpace` or just `Space`? Only the parent had `width: 100%`.

Reply:
Usually we want to force `width: 100%`, when for some reason this `<Space />` container (block) does not have width 100% by default. I checked each and every usage of this Antd's `<Space />` and we always want `<Space />` to be `width: 100%`. So basically forcing `width: 100%` at least for now does not harm anything